assets/store/User.js

```diff
@@ -155,6 +158,10 @@ export default class User extends Reactive {
     if (conversation && conversation[msg.dispatchTo]) conversation[msg.dispatchTo](msg);
   }
 
+  _shouldBeSilentMessage(msg) {
+    return ((msg.event == 'state') && this.ignoreStatuses);
```

Comment:
Oh. This was much simpler than what I had in mind. I guess you can just do this then:

```
msg.silent = this.ignoreStatuses;
// or
msg.ignoreStatuses = this.ignoreStatuses;
```